### DISCUSSION TITLE
Selected and multi-selected tab color

### DISCUSSION THREAD

**Discussion by manyfew** [piroor/treestyletab#3126] [Q&A]
I'm trying to change the color of selected/multi-selected tabs (holding down Ctrl key while clicking). At the moment it's a very faint blue and if I add the following code the faint blue is no longer displayed so I have no way to tell which tabs are part of the multi-selction:

`tab-item tab-item-substance {
  background: black;
}`

I've treid this but it didn't work:

`tab-item.selected tab-item-substance {
  background: blue;
}`

Any ideas how this could be accomplished? Thanks!

**Comment by irvinm** [comment]
For multi-select, just change the variable that TST uses for the color and\or transparency:

```
:root.sidebar { --multiselected-color: white; }
:root.sidebar { --multiselected-color-opacity: 0.5; }
```

**Reply by manyfew** [reply]
Excellent, it works! Sorry for the stupid question but if I use this it also works, is ".sidebar" strictly needed?

```
:root { --multiselected-color: white; }
:root { --multiselected-color-opacity: 0.5; }
```

**Reply by irvinm** [reply]
You can really do any scope you want to ... it is really just a precaution to be "more specific" vs. "less specific".

**Reply by manyfew** [reply]
I don't know much about CSS but I get what you mean. Thank you :)
